scripts/xmpro-website-scripts/scrape-xmpro-website-usecases.py

```diff
@@ -0,0 +1,147 @@
+import os
+import requests
+from bs4 import BeautifulSoup
+from pathvalidate import sanitize_filename
```

Comment:
@jacksonmcguirkxmpro 

I see we are using a library here but previously did some find replaces - lets figure the best option and then make consistent